src/api/article/controller.tests.js

```diff
@@ -9,41 +9,54 @@ import ArticleType from './type.model';
 const request = supertest.agent(app.listen());
 
 describe('Articles api', () => {
-  describe('# get all articles with pagination', () => {
-    before(async () => {
-      const articleType = await ArticleType.findOne({ name: 'Wir' });
-      const promises = [];
-      for (let i = 1; i < 9; i++) {
-        promises.push(
-          new Article({
-            title: `Api testing ${i} tit.`,
-            subtitle: `Api testing ${i} sub.`,
-            // eslint-disable-next-line no-underscore-dangle
-            type: articleType._id,
-            slug: `article-${i}`,
-            createdAt: new Date(`2015-01-0${i}T18:25:43.511Z`),
-          }).save()
-        );
-      }
-      await Promise.all(promises);
-    });
+  before(async () => {
+    // Population DB with article types.
+    const articleTypes = [{ name: 'Wir' }];
+    await Promise.all(
+      articleTypes.map(async articleType => new ArticleType(articleType).save())
```

Comment:
why array and promise all? it's for future cases with several types?

Reply:
In any case I'll prefer to simplify code to just:

Reply:
done.